tests/phpunit/api/v3/MailingTest.php

```diff
@@ -296,8 +296,7 @@ public function testMailerPreview(): void {
     $this->assertEquals("Hello $displayName", $previewResult['values']['subject']);
     $this->assertStringContainsString("This is $displayName", $previewResult['values']['body_text']);
     $this->assertStringContainsString("<p>This is $displayName.</p>", $previewResult['values']['body_html']);
-    $this->assertMatchesRegularExpression('!>Forward this email</a>!', $previewResult['values']['body_html']);
-    $this->assertMatchesRegularExpression('!<a href="([^"]+)civicrm/mailing/forward&amp;reset=1&amp;jid=&amp;qid=&amp;h=\w*">!', $previewResult['values']['body_html']);
+    $this->assertMatchesRegularExpression('!civicrm/mailing/optout&amp;reset=1&amp;jid=&amp;qid=&amp;h=\w*!', $previewResult['values']['body_html']);
```

Comment:
I diluted this test a bit, because the link is not in an href, but I don't think it makes any difference for what is being tested? (I noticed that the img tag for open-tracking formats url args differently than the `civicrm/mailing/optout`.